src/app/qgstipfactory.cpp

```diff
@@ -97,19 +97,18 @@ QgsTipFactory::QgsTipFactory() : QObject()
                         " move them around with the arrow keys. For accurate positioning use the"
                         " <strong>Position and Size</strong> section, which can be found in the"
                         " tab <strong>Item Properties -> Position and Size</strong>."
-                        " The other move tool (map canvas icon with right arrow) allows one"
-                        " to move the map content within a map frame."
+                        " The other move tool (a scrolled map icon with four oriented arrows)"
+                        " allows one to move the map content within a map frame."
```

Comment:
> Better would be "allows you to...."

That makes sense, in German the "one" construct is often used to circumvent the question if the user should be addressed with a formal you "Sie" or an informal you "Du".